src/algebra/algebra/quotient.lean

```diff
@@ -0,0 +1,119 @@
+import algebra.algebra.subalgebra.basic
+
+universes u v
+
+namespace algebra
+
+variables (R : Type u) (A : Type v) [comm_semiring R] [comm_ring A] [algebra R A]
+
+namespace quotient
+
+variable (I : ideal A)
+
+def mk : A →ₐ[R] A ⧸ I :=
+alg_hom.mk (ideal.quotient.mk I) rfl (map_mul _) rfl (map_add _) (λ _, rfl)
```

Comment:
I think this already exists, it's [`ideal.quotient.mkₐ`](https://leanprover-community.github.io/mathlib_docs/ring_theory/ideal/operations.html#ideal.quotient.mk%E2%82%90)

Reply:
You're right. Fixed in c7dfff30b.